docs/dev-proc/source-code-structure.md

```diff
@@ -4,41 +4,33 @@ Every repository forked and maintained by Dasharo Release Team has following
 branch structure:
 
 * `master` - follows upstream project master branch
-* `<platform>/release` - contains all code releases for given `<platform>`,
-  list of supported platforms is in
+* `dasharo` - contains all code releases for supported platforms, the list of
+   supported platforms is in
   [Hardware Compatibility List](../variants/hardware-compatibility-list.md)
   section
-* `<platform>/rel_vX.Y.Z` - release branch for version X.Y.Z
-* `<platform>/develop` - contains most recent development and is periodically
-  synced with `master` branch
-* `<platform>/<feature>` - tracks development of platform specific feature
+* `<platform>_rel_vX.Y.Z` - release branch for version X.Y.Z
```

Comment:
```suggestion
* `<platform>/rel_vX.Y.Z` - release branch for version X.Y.Z
```

The old scheme is good

Reply:
Although not sure if it will be needed

Reply:
fixed in 1ed0a83f08334d132029bbed91dd088fea7ee020